rllib/utils/replay_buffers/multi_agent_mixin_replay_buffer.py

```diff
@@ -344,7 +344,7 @@ def check_buffer_is_ready(_policy_id):
                     if check_buffer_is_ready(policy_id):
                         samples.append(mix_batches(policy_id).as_multi_agent())
 
-            return MultiAgentBatch.concat_samples(samples)
+            return concat_samples(samples) if samples else MultiAgentBatch({}, 0)
```

Comment:
I would prefer writing this part like the following, it's much simpler and also explicit about the output being a MultiAgentBatch type. 
```
concat_samples_into_ma_batch(samples or [])
```

Reply:
Also you didn't get rid of `MultiAgentBatch.concat_samples()` entirely in file. I found one that is slightly above this line. Might be worth doing a search and replace? I wonder why this is not caught on the unittests if the error on deprecation is raised.

Reply:
Mhhh that's weird. I'll check again, thanks! I'll follow the nits

Reply:
I looked again and didn't find any occurences. But some occurences of `concat_samples()` which I turned into `concat_samples_into_ma_batch()`! Lmk if you can point to the MultiAgentBatch.concat_samples() calls

Reply:
PyCharm also does not give me any results through search..

Reply:
Sorry, wrong call. :)